crates/ruff_linter/src/rules/flake8_gettext/rules/printf_in_gettext_func_call.rs

```diff
@@ -54,7 +54,7 @@ impl Violation for PrintfInGetTextFuncCall {
 pub(crate) fn printf_in_gettext_func_call(checker: &Checker, args: &[Expr]) {
     if let Some(first) = args.first() {
         if let Expr::BinOp(ast::ExprBinOp {
-            op: Operator::Mod { .. },
+            op: Operator::Mod,
```

Comment:
Interesting, is this now allowed or was it always allowed and I wasn't aware of it? i.e., matching against an enum struct variant without specifying the fields.